### DISCUSSION TITLE
update keda deployment to the latest version with helm (or another way)

### DISCUSSION THREAD

**Discussion by dariuszbz** [kedacore/keda#2512] [General] [answered]
Hello chops.

Perhaps the question is not valid, but I'm not a helm/keda expert: is it possible to update keda deployment i recently done with helm to the latest version from repo?
Let's say today i do install keda:

helm repo add kedacore https://kedacore.github.io/charts
helm repo update
helm install keda kedacore/keda

after few days I run:

helm repo update

and notice there is a Keda new version.

is there a magic trick: helm update keda kedacore/keda or any other magic spell?

I know I can uninstall and install again - but ... I'm looking for some more elegant solution. Thank You !

**Comment by tomkerkhove** [comment, marked as answer]
Yes, Helm supports this but this is out of scope for KEDA I'm afraid.

This should help you easily upgrade to the latest: (don't forget to `helm repo update` first)
https://helm.sh/docs/helm/helm_upgrade/

**Reply by dariuszbz** [reply]
Yes I was expectig that. Thank you very much. So pretty helm is:
helm upgrade keda kedacore/keda

**Comment by dariuszbz** [comment]
we can close that now. Thanks folks.
